app/controllers/base_controller.rb

```diff
@@ -3,9 +3,10 @@ class BaseController < ApplicationController
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  after_action :verify_authorized, except: :index
+  after_action :verify_authorized
 
```

Comment:
Unless I'm mistaken, there are no index actions currently. I am happy to add the action back in, but to me it feels like adding confusion to an already not very clear authorization procedure. Should we reevalute this when the refactoring of 'DefenceRequestController' is complete?